example/local/01-shoot.yaml

```diff
@@ -34,8 +35,15 @@ spec:
     #   enabled: true
     #   secretReferenceName: rsyslog-tls-certificates
     #   authMode: name # {name, fingerprint}
+    #   tlsLib: openssl # {openssl, gnutls}
```

Comment:
If `openssl` works everywhere, then we could consider using it internally instead of exposing it as configuration option. I don't know in which cases you (an end user) would need to specify `gnutls`? The docs (https://www.rsyslog.com/doc/v8-stable/configuration/modules/imrelp.html#tls-tlslib) recommend using `openssl`.

Reply:
The problem is that `openssl` doesn't work everywhere :(. It depends on what library was used to generate the certificates, what tls library is currently available on the system, the version of `librelp` and probably other factors.